gluoncv/model_zoo/attention.py

```diff
@@ -0,0 +1,71 @@
+import mxnet as mx
+from mxnet.gluon import nn
+from mxnet.gluon.nn import HybridBlock
+# pylint: disable-all
+
+__all__ = ['PAM_Module', 'CAM_Module']
+
+
+class PAM_Module(HybridBlock):
+    """ Position attention module"""
+    #Ref from SAGAN
```

Comment:
Add more descriptions on what is a PAM module and a CAM module. If this is proposed in a paper, please put the paper title and link here.